### PR TITLE
all of index pages made responsive

### PR DESCRIPTION
**Pull request by Ramadan1011**

### CODE REVIEW

**Review by jamshidbek7375 — DISMISSED** [review]

**Review by Boburm1rzo — DISMISSED** [review]

**Comment by BegzhanMaxamatxanov** [comment]
Nice!

**Review by BegzhanMaxamatxanov — APPROVED** [review]
Good job)

File: ExpenseTracker/ExpenseTracker/Views/Shared/_Sidebar.cshtml [hunk 1]
```diff
@@ -80,10 +79,33 @@
...
+                    dockBar.toggle();
+                    mainContent.classList.add('sidebar-collapsed');
+                }
+            } else {
+                if (!dockBar.isOpen) {
+                    dockBar.toggle();
+                    mainContent.classList.remove('sidebar-collapsed');
+                }
+            }
+        }
```
Comment (davron41) [inline ExpenseTracker/ExpenseTracker/Views/Shared/_Sidebar.cshtml:108]: if (creenWidth < 768 && dockBar.isOpen) {
  // logic
  return;
}

if (!dockBar.isOpen){
  // logic
}

**Review by davron41 — CHANGES_REQUESTED** [review]
please, perform small code quality improvements.

**Review by jamshidbek7375 — APPROVED** [review]

**Review by Boburm1rzo — APPROVED** [review]
👍

**Review by davron41 — APPROVED** [review]
looks good.

File: ExpenseTracker/ExpenseTracker/Views/Shared/_Sidebar.cshtml (hunk 1 above)
Comment (Ramadan1011) [reply ExpenseTracker/ExpenseTracker/Views/Shared/_Sidebar.cshtml]: fixed

**Pull request merged**